deployers/servicedeployer.go

```diff
@@ -412,7 +412,7 @@ func (deployer *ServiceDeployer) createFeedAction(trigger *whisk.Trigger, feedNa
 		Publish:     &pub,
 	}
 
-	_, _, err := deployer.Client.Triggers.Insert(t, false)
+	_, _, err := deployer.Client.Triggers.Insert(t, true)
```

Comment:
Defaulting to override=true as the default matches all the other Insert() calls for all other entities.